src/frontend/src/features/rooms/api/ApiRoom.ts

```diff
@@ -8,4 +8,7 @@ export type ApiRoom = {
     room: string
     token: string
   }
+  configuration?: {
+    [key: string]: string | number | boolean;
```

Comment:
didn't quite get what the configuration could contain just reading the code. Is this an object with specific keys that we should list here or…?

Reply:
Json field in the backend, not format specified yet.